src/lib/components/Banner.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+	export let message = 'Welcome to our website!'
+	export let bgColor = '#4CAF50' // Default background color
+
+	let visible = true // Initially, the banner is visible
+</script>
+
+{#if visible}
+	<div class="banner" style="--bg-color: {bgColor};">
+		<span>{@html message}</span>
+		<button class="close-btn" on:click={() => (visible = false)}>×</button>
+	</div>
+{/if}
+
+<style>
+	.banner {
+		padding: 16px;
```

Comment:
In general, try to use `rem` for all units. Media queries are an exception.

Reply:
done